l1-contracts/test/validator-selection/ValidatorSelection.t.sol

```diff
@@ -195,11 +100,22 @@ contract ValidatorSelectionTest is DecoderBase {
     uint256 expectedSize =
       validatorSetSize > targetCommitteeSize ? targetCommitteeSize : validatorSetSize;
 
-    assertEq(rollup.getEpochCommittee(pre).length, expectedSize, "Invalid committee size");
-    assertEq(rollup.getEpochCommittee(post).length, expectedSize, "Invalid committee size");
+    address[] memory preCommittee = rollup.getEpochCommittee(pre);
+    address[] memory postCommittee = rollup.getEpochCommittee(post);
+    assertEq(preCommittee.length, expectedSize, "Invalid committee size");
+    assertEq(postCommittee.length, expectedSize, "Invalid committee size");
+
+    // Elements in the committee should be the same
+    for (uint256 i = 0; i < expectedSize; i++) {
```

Comment:
Don't forgestd come with list comparison actually? Then we can get rid of a bunch of lines here and it is very clear then 🤷

Reply:
Ah, left over from when i was logging them, fixing